src/Servers/IIS/IIS/test/IIS.ShadowCopy.Tests/DeployerSelector.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Suggest using a single copy of this `DeployerSelector` and compiling it into the projects that use the same selections

Reply:
I'm going to leave these for now, but keep it in mind for the future, I don't love these DeployerSelector psuedo bit flag files either

Reply:
I'll include the DeployerSelector in the cleanup workitem for improving these functional tests